pynest/nest/lib/hl_api_simulation.py

```diff
@@ -190,99 +190,134 @@ def SetKernelStatus(params):
 
     Other parameters
     ----------------
-    The following parameters can be written to the kernel status dictionary.
+    The following parameters are available from the kernel status dictionary.
+    This is a list of all kernel parameters, the keys flagged with
+    `(read only)` are internally calculated and cannot be user defined.
+    Their values can be consulted using GetKernelStatus.
 
-    resolution : double
+    Time and resolution
```

Comment:
@sarakonradi Should headings such as this be marked up in RST syntax and if so, using which symbol?

Reply:
It would be a good idea to stick to the style we have used for .py files previously, such as those in ``pynest/examples``. There, the main headings have a dashed line. The subsections (in this case "Time and resolution") use tildes. There also needs to be a blank line after the characters: 

    Other parameters
    ----------------

    The following parameters are available from the kernel status dictionary.
    This is a list of all kernel parameters, the keys flagged with
    `(read only)` are internally calculated and cannot be user defined.
    Their values can be consulted using GetKernelStatus.

    Time and resolution
    ~~~~~~~~~~~~~~~~~~~

    resolution : float

Reply:
Hi, I was looking at this and have one question. Do you leave a blank line between all headings and text? Or can I just write immediately after?

Reply:
Fixed to a blank line is needed after talking to @sarakonradi

Reply:
@morales-gregorio and I discussed this offline (see comment below)